test/sql/index/art/test_art_index_range_scan.test

```diff
@@ -0,0 +1,27 @@
+# name: test/sql/index/art/test_art_index_range_scan.test
+# description: Test ART index scan
+# group: [art]
```

Comment:
Is it necessary to commit this test? 
We do have other range tests running already, so this test doesn't add any coverage increase (i.e., not a test that was breaking on master, but your fix is rather on an inefficiency).